mock-mas-api/src/main/java/gov/va/vro/mockmas/controller/MockMasController.java

```diff
@@ -38,6 +42,8 @@ public class MockMasController {
   private final MasApiService apiService;
   private final CollectionStore store;
 
+  @Autowired private final ExamOrderStore examOrderStore;
```

Comment:
You do not need @Autowired here. "private final" will cause constructor injection since we have the @RequiredArgsConstructor as well. Constructor injection is preferred.